templates/spring-mvc-maven-mongodb/files/.platform.app.yaml

```diff
@@ -26,14 +26,4 @@ relationships:
 # The configuration of app when it is exposed to the web.
 web:
     commands:
-        start:  |
-           export USER=`echo $PLATFORM_RELATIONSHIPS|base64 -d|jq -r ".database[0].username"`
-           export PASSWORD=`echo $PLATFORM_RELATIONSHIPS|base64 -d|jq -r ".database[0].password"`
-           export HOST=`echo $PLATFORM_RELATIONSHIPS|base64 -d|jq -r ".database[0].host"`
-           export DATABASE=`echo $PLATFORM_RELATIONSHIPS|base64 -d|jq -r ".database[0].path"`
-           java -jar -Xmx$(jq .info.limits.memory /run/config.json)m -XX:+ExitOnOutOfMemoryError \
-           -Dspring.data.mongodb.database=$DATABASE \
-           -Dspring.data.mongodb.host=$HOST \
-           -Dspring.data.mongodb.username=$USER \
-           -Dspring.data.mongodb.password=$PASSWORD \
-            target/spring-mvc-maven-mongodb.jar --server.port=$PORT
+        start: java -jar $JAVA_OPTS $CREDENTIAL target/spring-mvc-maven-mongodb.jar --server.port=$PORT
```

Comment:
Why move all of these to a separate file? That seems like it makes it harder to read since it's spread out now.